app/controller/InfoController.js

```diff
@@ -101,13 +102,17 @@ SDL.InfoController = Em.Object.create(
           app_title = app['nicknames'][0];
         }
 
+        const icon_url = app.hasOwnProperty('icon_url') ? app['icon_url'] : 'images/info/info_leftMenu_apps_ico.png';

```

Comment:
```suggestion
        const icon_url = ('icon_url' in app) ? app['icon_url'] : 'images/info/info_leftMenu_apps_ico.png';
```

`in` is much faster than `hasOwnProperty`

Reply:
@iCollin fixed in https://github.com/smartdevicelink/sdl_hmi/pull/274/commits/b506afc2d807d2ffa04860513272c8f6129f7f03